### PR TITLE
Don't allow blank adhoc command module names

### PR DESCRIPTION
**Pull request by jakemcdermott**
##### SUMMARY

### CODE REVIEW

File: awx/api/serializers.py [hunk 1]
```diff
@@ -3204,7 +3204,7 @@ def build_standard_field(self, field_name, model_field):
             field_kwargs['choices'] = module_name_choices
             field_kwargs['required'] = bool(not module_name_default)
             field_kwargs['default'] = module_name_default or serializers.empty
-            field_kwargs['allow_blank'] = bool(module_name_default)
+            field_kwargs['allow_blank'] = False
```
Comment (jakemcdermott) [inline awx/api/serializers.py:3208]: My rationale for this change is that you really do need a module name to run an AHC. It also fixes this:

Screenshot from 2020-03-25 11-37-09

File: awx/api/serializers.py [hunk 2]
```diff
@@ -3204,7 +3204,7 @@ def build_standard_field(self, field_name, model_field):
             field_kwargs['choices'] = module_name_choices
             field_kwargs['required'] = bool(not module_name_default)
             field_kwargs['default'] = module_name_default or serializers.empty
```
Comment (jakemcdermott) [inline awx/api/serializers.py:3207]: For similar reasons as my other comment, I'm guessing this maybe should just be `field_kwargs['default'] = module_name_default` ?

Comment (jbradberry) [reply awx/api/serializers.py:3207]: Based on what I see, I believe the point of the original machinations here is for the `command` module to be the implicit default, if it is in the choices list provided by `settings.AD_HOC_COMMANDS`.

**Comment by jakemcdermott** [comment]
nbd
recheck

**Comment by jakemcdermott** [comment]
recheck

**Comment by jakemcdermott** [comment]
:middle_finger: :middle_finger: :middle_finger: recheck :middle_finger: :middle_finger: :middle_finger:

**Comment by jakemcdermott** [comment]
I did a full run with internal tooling on this PR and any failures listed were prior ones.

**Review by ryanpetrello — APPROVED** [review]

**Pull request merged**